wiki/Glossary/Ignore_List/en.md

```diff
@@ -0,0 +1,12 @@
+---
+tags:
+  - blacklist
+---
+
+# Ignore List
+
+The **ignore list** is a blacklist of words (or users) that the player does not want to be displayed in the [chat console](/wiki/chat_console). It can be accessed and modified in the [user settings menu](https://osu.ppy.sh/home/account/edit) under the privacy tab.
```

Comment:
```suggestion
The **ignore list** is a blacklist of words (or users) that a player does not want to be displayed in the [chat console](/wiki/chat_console). The ignore list can be accessed and modified in the [user settings menu](https://osu.ppy.sh/home/account/edit) under the privacy tab.
```
"It" may make this too ambiguous as it could be mistaken as referring to the chat console.